nixos/modules/config/adwaita-qt.nix

```diff
@@ -0,0 +1,30 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+{
+  options = {
+    adwaita-qt.enable = mkOption {
```

Comment:
Yeah not sure about having this option path. Maybe somewhere in `services.xserver` would be a better place

Reply:
We could think of it as a theme, though not completely. But I don't think `services.xserver` would make sense because it's not a service.

`programs.adwaita-qt` isn't right either, I think I might have to stick with it like this.

Reply:
Maybe we could introduce a `misc` grouping for `misc.adwaita-qt.enable`

Reply:
But then it would mirror the kinda messy hierarchy we have for packages.
Maybe it doesn't have to be namespaced?

Reply:
Well with something like `misc` at least the mess won't be interleaved with the structured parts.

Maybe `theming.adwaita-qt.enable`?

Reply:
I've just realized that the correct module is in home-manager
* https://github.com/rycee/home-manager/blob/95d55b8da152f1efffb7713c82ead83a3fcf476d/modules/misc/qt.nix#L29

This just needs a new option `style` so we can set the Qt style to `adwaita-qt`.

so
* `qt.platformTheme`
* `qt.style`